File: UML/class/Skill.ts
export abstract class Skill {
  private name: string;
  private description: string;

  constructor(name: string, description: string) {
    this.name = name;
    this.description = description;
  }

  public getNanem(): string {
    return this.name;
  }

  public getDescription(): string {
    return this.description;
  }
}

File: Project1/Skill.cpp
#include "Skill.h"

Skill::Skill(std::string_view name, std::string_view description) {
    this->name = name;
    this->description = description;
}

std::string Skill::getName() {
    return name;
}

std::string Skill::getDescription() {
    return description;
}
